### PR TITLE
feat: update APKs of only corresponding branches when updated

### LINKED ISSUE #378: Update apk script
- [ ] When PRs are merged in the dev branch, add apks to the apk branch. Include “dev” in the name

 - [ ] When PRs are merged in the master branch, add apks using the master branch to the apk branch

 - [ ] Dev apks should only be updated when PRs are merged to dev. Master apks should only be updated when PRs are merged to Master.

### PR DESCRIPTION
**Pull request by simarsingh24**
fixes #378

### CODE REVIEW

**Comment by simarsingh24** [comment]
@iamareebjamal is this right we need to add following things
 - [ ] When PRs are merged in the dev branch, add apks to the apk branch. Include “dev” in the name

 - [ ] When PRs are merged in the master branch, add apks using the master branch to the apk branch

 - [ ] Dev apks should only be updated when PRs are merged to dev. Master apks should only be updated when PRs are merged to Master.

**Comment by iamareebjamal** [comment]
No matter what you name the apks, third point can never be achieved

File: scripts/update-apk.sh [hunk 1]
```diff
@@ -17,9 +17,9 @@ git clone --quiet --branch=apk https://fossasia:$GITHUB_API_KEY@github.com/fossa
 cd apk
 
 if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
-	/bin/rm -f *
+	/bin/rm -f open-event-master-app-debug.apk open-event-master-app-release-unsigned.apk
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: Wrong app names, thers's fdroid and play store version

File: scripts/update-apk.sh [hunk 2]
```diff
@@ -35,6 +35,12 @@ if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
 	done
 fi
 
+if [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
+	for file in app*; do
+		cp $file open-event-dev-${file%%}
+	done
+fi
+
```
Comment (simarsingh24) [inline scripts/update-apk.sh]: @iamareebjamal we have renamed the apks generated from deploy branch, thats why i renamed it above

Comment (iamareebjamal) [reply scripts/update-apk.sh]: You have not appended build flavor in the apk name

Comment (simarsingh24) [reply scripts/update-apk.sh]: wouldnt `$file` will add the build flavor like` app-debug` or `app-release`?

Comment (iamareebjamal) [reply scripts/update-apk.sh]: These are the file names you have written above `open-event-master-app-debug.apk open-event-master-app-release-unsigned.apk ` 

See any problem with these?

File: scripts/update-apk.sh (hunk 1 above)
Comment (simarsingh24) [reply scripts/update-apk.sh]: here open-evet-master should be changed to open-event-dev right?

Comment (iamareebjamal) [reply scripts/update-apk.sh]: No

File: scripts/update-apk.sh [hunk 3]
```diff
@@ -17,9 +17,9 @@ git clone --quiet --branch=apk https://fossasia:$GITHUB_API_KEY@github.com/fossa
 cd apk
 
 if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
-	/bin/rm -f *
+	/bin/rm -f open-event-master-app-debug.apk open-event-master-app-release-unsigned.apk 
 else
-	/bin/rm -f app-fdroid-debug.apk app-playStore-debug.apk app-playStore-release.apk app-fdroid-release.apk
+	/bin/rm -f open-event-master-app-fdroid-debug.apk open-event-master-app-playStore-debug.apk open-event-master-app-playStore-release.apk open-event-master-app-fdroid-release.apk
```
Comment (simarsingh24) [inline scripts/update-apk.sh]: sorry here open-event-master should be changed to open-event-dev

File: scripts/update-apk.sh (hunk 1 above)
Comment (iamareebjamal) [reply scripts/update-apk.sh]: My concern is that the names you have written here are wrong

File: scripts/update-apk.sh (hunk 3 above)
Comment (iamareebjamal) [reply scripts/update-apk.sh]: Yes

File: scripts/update-apk.sh [hunk 4]
```diff
@@ -35,6 +35,12 @@ if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
 	done
 fi
 
+if [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
+	for file in app*; do
+		cp $file open-event-dev-${file%%}
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: This will also rename `open-event-master` apks

**Comment by simarsingh24** [comment]
@iamareebjamal will this work?

File: scripts/update-apk.sh (hunk 1 above)
Comment (iamareebjamal) [reply scripts/update-apk.sh]: Still wrong

Comment (simarsingh24) [reply scripts/update-apk.sh]: is it right now?

File: scripts/update-apk.sh [hunk 5]
```diff
@@ -17,7 +17,7 @@ git clone --quiet --branch=apk https://fossasia:$GITHUB_API_KEY@github.com/fossa
 cd apk
 
 if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
-	/bin/rm -f open-event-master-app-debug.apk open-event-master-app-release-unsigned.apk 
+	/bin/rm -f  open-event-master-app-playStore-release.apk open-event-master-app-fdroid-release.apk
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: And debug versions?

Comment (simarsingh24) [reply scripts/update-apk.sh]: added

File: scripts/update-apk.sh [hunk 6]
```diff
@@ -31,10 +31,22 @@ fi
 
 if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
 	for file in app*; do
+		if [[ $file = *"dev"* ]]; then
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: The logic should be that if flle name starts from `open-event`, skip it

**Pull request merged**